packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -169,8 +169,22 @@ export class ListItem extends LitElement implements InteractiveComponent, Sortab
    */
   @property() moveToItems: MoveTo[] = [];
 
-  /** When `true`, the item is open to show child components. */
-  @property({ reflect: true }) open = false;
+  /**
+   * When `true`, the item is open to show child components.
+   *
+   * @deprecated Use `expanded` prop instead.
+   */
+  @property({ reflect: true })
+  get open(): boolean {
+    return this.expanded;
+  }
+
+  set open(value: boolean) {
+    this.expanded = value;
```

Comment:
Side note: I think we can start using getters/setters to warn on deprecated prop usage (using [`logger.deprecated`](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/utils/logger.ts#L64-L72)). Could you create an issue for this?

Reply:
[Add deprecation warnings for deprecated properties](https://github.com/Esri/calcite-design-system/issues/11653#top) #11653